tests/system/test_grpc_streams.py

```diff
@@ -71,3 +75,145 @@ def test_stream_stream_passing_dict(echo):
     assert contents == ['hello', 'world!']
 
     assert responses.trailing_metadata() == metadata
+
+
+@pytest.mark.asyncio
+async def test_async_unary_stream_reader(async_echo):
+    logging.debug('test_async_unary_stream_reader %s %s',
+                  threading.current_thread(), asyncio.get_event_loop())
+    content = 'The hail in Wales falls mainly on the snails.'
+    call = await async_echo.expand({
+        'content': content,
+    }, metadata=metadata)
+    logging.debug('test_async_unary_stream_reader 2')
```

Comment:
Are these additional numbered debug statements left in this function intentionally?